modules/ROOT/pages/mongodb/mongodb-connector-6-0-studio.adoc

```diff
@@ -0,0 +1,100 @@
+= MongoDB Studio Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+MongoDB Connector Version 6.0
+
+To configure a connector in Anypoint Studio follow this process:
+
+. Install the connector.
+. Configure an input source for the connector.
+. Configure the connector.
+
+== Install the Connector Using Exchange
+
+. In Studio, create a Mule project.
+. Click the Exchange *(X)* icon in the upper left of the Studio task bar.
+. In Exchange, click *Login* and supply your Anypoint Platform username and password.
+. In Exchange, search for `MongoDB`.
+. Select the MongoDB Connector and click *Add to project*.
+. Follow the prompts to install the connector.
+
+== Install the Connector in Studio
+
+. In Studio, create a Mule project.
+. In the Mule Palette, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, type the name of the connector in the search field.
+. Click the MongoDB Connector name in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+== Configure an Input Source for the Connector
```

Comment:
Sometimes we say Source or Listener or Trigger but we must always use the same word for it. It applies for the entire platform

Reply:
This is an issue. We are now saying input source for Studio and trigger for Flow Designer. Flow Designer uses the term trigger. We relate the two terms in the Flow Designer topic. I will check to see if there are instances where these terms and listener are not used correctly.